security/windef_entries.ksy

```diff
@@ -0,0 +1,242 @@
+                                           0xB9, 0xF2, 0xA2, 0x2E, 0xD1, 0xE9, 0xE1, 0x1D, 0x75, 0xBE,
+                                           0xD7, 0xDC, 0x0E, 0xCB, 0x0A, 0x8E, 0x68, 0xA2, 0xFF, 0x12,
+                                           0x63, 0x40, 0x8D, 0xC8, 0x08, 0xDF, 0xFD, 0x16, 0x4B, 0x11,
+                                           0x67, 0x74, 0xCD, 0x0B, 0x9B, 0x8D, 0x05, 0x41, 0x1E, 0xD6,
+                                           0x26, 0x2E, 0x42, 0x9B, 0xA4, 0x95, 0x67, 0x6B, 0x83, 0x98,
+                                           0xDB, 0x2F, 0x35, 0xD3, 0xC1, 0xB9, 0xCE, 0xD5, 0x26, 0x36,
+                                           0xF2, 0x76, 0x5E, 0x1A, 0x95, 0xCB, 0x7C, 0xA4, 0xC3, 0xDD,
+                                           0xAB, 0xDD, 0xBF, 0xF3, 0x82, 0x53
+                                          ])
+types:
+  rc4encrypted_header:
+    seq:
+      - id: magic
+        size: 0x10
+        contents: [0xdb, 0xe8, 0xc5, 0x01, 0x01, 0, 0x01, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+      - id: unknown1
+        size: 0x18
+      - id: len1
+        type: u4
+      - id: len2
+        type: u4
+      - id: unknown2
+        size: 0x0C
+  encrypted_data1:
+    seq:
+      - id: guid
+        type: guid
+      - id: unknown3
+        size: 16
+      - id: time
+        type: winfiletime
+      - id: id11
+        type: u4
+      - id: id12
+        type: u2
+      - id: id13
+        type: u2
+      - id: number_of_strings
+        type: u4
+      - id: mal_type
+        type: str
+        encoding: UTF-8
+        terminator: 0
+  encrypted_data2:
+    seq:
+      - id: number_of_entries
+        type: u4
+      - id: offset
+        type: u4
+        repeat: expr
+        repeat-expr: number_of_entries
+    instances:
+      entries:
+        pos: offset[0]
+        repeat: expr
+        repeat-expr: number_of_entries
+        type:
+          switch-on: _index != number_of_entries-1
+          cases:
+            true: entrys(_index)
+            false: entrye
+    types:
+      entrys:
+        params:
+          - id: i
+            type: u4
+        seq:
+          - id: entry
+            size: _parent.offset[i + 1] - _parent.offset[i]
+            type: entry
+      entrye:
+        seq:
+          - id: entry
+            type: entry
+  entry:
+    seq:
+      - id: path
+        type: null_terminated_utf16le
+      - id: number_of_elements
+        type: u2
+      - id: typestr
+        type: str
+        encoding: UTF8
+        terminator: 0x00
+      - id: padding
+        size: (4 - _io.pos) % 4
+      - id: element
+        type: listelement
+        repeat: expr
+        repeat-expr: number_of_elements
+  null_terminated_utf16le:
+    seq:
+      - id: character
+        type: u2
+        repeat: until
+        repeat-until: character[_index] == 0x0000
+  listelement:
+    seq:
+      - id: length
+        type: u2
+      - id: elementsubtype
+        type: u1
```

Comment:
are these enums?

Reply:
I don't know what this field is doing, but it seems to describe somehow what this element is. However, I only saw that only "elementtype" seems to be relevant for parsing.